fiftyone/utils/hunyuan3d.py

```
@@ -0,0 +1,230 @@
+                    mesh_path = os.path.join(output_dir, mesh_filename)
+                    mesh.export(mesh_path)
+
+                    outputs.append(
+                        fol.Label(
+                            label="3d_mesh",
+                            mesh_path=mesh_path,
+                            vertices=mesh.vertices.shape[0],
+                            faces=mesh.faces.shape[0],
+                        )
+                    )
+                except (OSError, IOError, ValueError, RuntimeError) as e:
```

Ruff (0.14.10)

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)